students/hyein/users/views.py

```diff
@@ -52,12 +48,21 @@ def post(self, request):
             email    = data['email']
             password = data['password']
             
-            if User.objects.filter(email = email, password = password).exists():
-                return JsonResponse({'Message': 'SUCCESS'}, status=200)
-            return JsonResponse({'Message': 'INVALID_USER'}, status=401)
+            user = User.objects.get(email = email)
+            
+            if bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')) == False:
+                return JsonResponse({'Message': 'Invalid Password'}, status=401)
```

Comment:
파이썬 PEP 8 프로그래밍 권고에는 비교문 관련해서 어떤 형식을 사용하라고 안내되어 있습니다.

- True/False를 비교할 때는 if문에 변수만 사용하도록 한다.
- 동등성(단순히 값만 비교하는 경우)을 검증할 때는 == 혹은 != 같은 연산자를 사용하도록 한다.
- 동일성(객체의 고유 메모리 주소로 비교하는 경우)을 검증할 때는 is나 is not을 사용한다.

즉 아래의 경우는 True/False만을 비교하기 때문에 `== False`는 붙일 필요가 없습니다.
```suggestion
            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
                return JsonResponse({'Message': 'Invalid Password'}, status=401)
```